pinot-controller/src/main/resources/app/components/Homepage/Operations/RebalanceServer/RebalanceResponse.tsx

```diff
@@ -0,0 +1,63 @@
+import {RebalanceServerSection} from "./RebalanceServerSection";
+import React from 'react';
+import {Grid, Paper} from "@material-ui/core";
+import CustomCodemirror from "../../../CustomCodemirror";
+import {
+    RebalanceServerSectionResponse
+} from "./RebalanceServerResponses/RebalanceServerSectionResponse";
+import {RebalanceServerPreChecksResponse} from "./RebalanceServerResponses/RebalanceServerPreChecksResponse";
+import {RebalanceServerResponseLabelValue} from "./RebalanceServerResponses/RebalanceServerResponseLabelValue";
+import {
+    RebalanceServerRebalanceSummaryResponse
+} from "./RebalanceServerResponses/RebalanceServerRebalanceSummaryResponse";
+import {RebalanceServerResponseCard} from "./RebalanceServerResponses/RebalanceServerResponseCard";
+
+export const RebalanceResponse = ({ response }) => {
+    const responseSectionsToShow = [
+        {
+            name: 'Segment Assignment',
+            key: 'segmentAssignment'
+        },
+        {
+            name: 'Instance Assignment',
+            key: 'instanceAssignment'
+        }
+    ];
+
```

Comment:
@somandal for adding new sections you just need to provide the `key` and `name of the section` and it will render